manifests/rule.pp

```diff
@@ -64,48 +67,59 @@
 #   If undefined it defaults to the setting in logrotate
 # @param size
 # @param sharedscripts
+# @param shred
+# @param shredcycles
+# @param su
```

Comment:
a description of the su params would be nice as they do not one-for-one match a configuration item